### ISSUE TITLE
How to Execute code in window- OpenLCA Developer tools

### ISSUE THREAD

**Issue by priyaII** [GreenDelta/openlca-python-tutorial#1]
Hello @msrocka ! Thanks for putting this up :)

How do i execute the "say hello" code written in the python window in OpenLCA?

![image](https://cloud.githubusercontent.com/assets/20396800/26635940/e1d89160-4638-11e7-9f7d-6d5ed9293cb3.png)

Version - 1.6.2
Windows 7, 64bit comp

**Comment by msrocka** [comment]
Right, this is a bit hidden: you click on the green arrow in the toolbar (next to the save icons). Also, you first have to open a database in order to execute this script which is not very intuitive. I created an issue for this here: https://github.com/GreenDelta/olca-app/issues/23
